app/models/activity.rb

```diff
@@ -0,0 +1,4 @@
+# Activity model for customisation & custom methods
+class Activity < PublicActivity::Activity
+  has_many :notifications, inverse_of: :activity, dependent: :destroy
```

Comment:
I still do not understand why one activity can have many notifications. That sounds weird.